upload/src/addons/SV/WarningImprovements/XF/Repository/Warning.php

```diff
@@ -64,18 +68,40 @@ public function getSvWarningReplaceables(UserEntity $user, WarningEntity $warnin
                 $replacables['{' . $key . '}'] = $value;
             }
             $params = $replacables;
+            $params['content_action_options'] = $contentActionOptions;
         }
         else
         {
             foreach ($params as $key => &$value)
             {
+                if ($key === 'content_action')
+                {
+                    $value = $this->getContentActionForPhrase($value, $contentActionOptions);
```

Comment:
This logic should be hoisted out of the loop, and instead be conditionally adding the to `$params` key-value-pair before this if-statement.